CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+
+## Contributing
+
+We welcome all contributions!  That said, this is a Sage Bionetworks owned project, and we use JIRA ([AG](https://sagebionetworks.jira.com/jira/software/c/projects/AG/boards/91)/[IBCDPE](https://sagebionetworks.jira.com/jira/software/c/projects/IBCDPE/boards/189)) to track any bug/feature requests. If you are a Sage Bio employee, make sure to assign yourself the JIRA ticket if you decide to work on it.
+
+## Coding Style
+
+The code in this package is also automatically formatted by `black` for consistency.
+
+## The Development Life Cycle
+
+### Fork and clone this repository
+
+1. See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
```

Comment:
I thought we weren't working on forks in this repo?

Reply:
good catch, I will fix that.

Reply:
iirc, CI will fail if we get a PR from a fork because the fork doesn't have access to the required secrets (or something along those lines).

Reply:
What we could do is just explicitly state the contribution guidelines are specific to sage employees that work on this package. 

Once we figure out a "better" way of doing post processing validation, we may not need to have the synapse login.